### DISCUSSION TITLE
Find by Image, return only one result

### DISCUSSION THREAD

**Discussion by Arthurus1st** [appium/appium#15297] [Q&A]
Hi Everyone,
I'm using multiple client for Appium, Python, Java, PHP. Using the find_elements_by_image (Python client) I was expecting to have a list of elements.
Looking deeper into the code I did run into the bellow comment on the python client:
        Uses driver.find_image_occurrence under the hood. Note that this will
        only ever return at most one element
Is it an Appium limitation?
Thanks

**Comment by KazuCocoa** [comment]
Yes, until 1.20.2
The next one will return multiple images if it can find. You can try it out with `appium@rc` or `appium@beta`
https://github.com/appium/appium-base-driver/pull/449

**Comment by Arthurus1st** [comment]
Awesome Thanks KazuCocoa
